docs/testing-guide/ai-foundry-integration.md

```diff
@@ -1,6 +1,8 @@
 # Deploying SQL MCP Server implemented in Data API builder and Integrating with Azure AI Foundry
 
-This document provides an end‑to‑end guide to stand up a **SQL MCP Server** with **Model Context Protocol (MCP)** tools implemented in **Data API builder (DAB)** container that also exposes **REST** and **GraphQL** endpoints, and to integrate those MCP tools with an **Azure AI Foundry Agent**. 
+This document provides an end‑to‑end guide to stand up a **SQL MCP Server** with **Model Context Protocol (MCP)** tools implemented in **Data API builder (DAB)** container that also exposes **REST** and **GraphQL** endpoints, and to integrate those MCP tools with an **Azure AI Foundry Agent**.
+
+<img width="2398" height="2178" alt="image" src="https://github.com/Azure/data-api-builder/blob/95cd0c1f83b3caa8fcafec64d4c399d9c58e31e2/docs/media/Architecture.png" />
```

Comment:
the diagram reference still points to a blob url. You can simply provide a relative reference to `../media/dab-aifoundry-architecture.png`